### ISSUE TITLE
Typo: root_dir vs rocks_dir

### ISSUE THREAD

**Issue by daurnimator** [keplerproject/luarocks#278]
One of the following is incorrect:

https://github.com/keplerproject/luarocks/blob/8498916df098bfcf17947537427cc1fffdeea2a5/src/luarocks/manif.lua#L417

``` lua
-- @param repo string or nil: Pathname of a local repository. If not given,
-- the default local repository configured as cfg.rocks_dir is used.
```

https://github.com/keplerproject/luarocks/blob/8498916df098bfcf17947537427cc1fffdeea2a5/src/luarocks/manif.lua#L426

``` lua
repo = path.rocks_dir(repo or cfg.root_dir)
```

**Comment by daurnimator** [comment]
Might be another one: https://github.com/keplerproject/luarocks/blob/master/src/luarocks/remove.lua#L162 ?

I'm getting:
```
Error: LuaRocks 2.2.0 bug (please report at luarocks-developers@lists.sourceforge.net).
/usr/share/lua/5.1/luarocks/remove.lua:162: attempt to concatenate field 'root_dir' (a table value)
stack traceback:
	/usr/share/lua/5.1/luarocks/remove.lua:162: in function </usr/share/lua/5.1/luarocks/remove.lua:140>
	(tail call): ?
	[C]: in function 'xpcall'
	/usr/share/lua/5.1/luarocks/command_line.lua:208: in function 'run_command'
	/usr/bin/luarocks-5.1:32: in main chunk
	[C]: ?
```

**Comment by ignacio** [comment]
This was fixed in db46b2203d2348c3b470605430cdb183466e259e.
